### PR TITLE
Dev mandeep

### PR DESCRIPTION
**Pull request by Mandeep9952**
A project using two APIs 
1. OpenWeatherApi
2. Geobytes for all city names

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,10 +33,10 @@ <h1 class="jumbotron-heading">Projects List</h1>
                         <a href="credits.html" class="btn btn-primary">Main Project</a>
                     </p>
                     <p>
-                        <a href="#" class="btn btn-primary">Project 01</a>
+                        <a href="mandeep.html" class="btn btn-primary">Open Weather API</a>
                     </p>
                     <p>
-                        <a href="#" class="btn btn-primary">Project 02</a>
```
Comment (caiodias) [inline index.html]: Why are you using two buttons on the root index.html file?

**Comment by caiodias** [comment]
Please organize your project files under a folder starting with your GitHub user. Otherwise it will be a nightmare for me to merge all projects.

File: index.html (hunk 1 above)
Comment (Mandeep9952) [reply index.html]: sorry i forget to remove it
i  will fix it

**Comment by caiodias** [comment]
Well done

**Comment by Mandeep9952** [comment]
Thank you.

**Pull request merged**